bigtable/admin.go

```diff
@@ -604,6 +603,13 @@ func (ac *AdminClient) TableIAM(tableID string) *iam.Handle {
 		"projects/"+ac.project+"/instances/"+ac.instance+"/tables/"+tableID)
 }
 
+// BackupIAM creates an IAM client specific to a given Cluster and Backup.
```

Comment:
```suggestion
// BackupIAM creates an IAM Handle specific to a given Cluster and Backup.
```

Reply:
Should make this correction in L600 above as well.